test/e2e/serial/tests/workload_placement_tmpol.go

```diff
@@ -522,7 +522,7 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload placeme
 			Expect(err).ToNot(HaveOccurred())
 
 			// the NRT updaters MAY be slow to react for a number of reasons including factors out of our control
-			// (kubelet, runtime). This is a known behaviour. We can only tolerate some delay in reporting on pod removal.
+			// (kubelet, runtime). This is a known behavior. We can only tolerate some delay in reporting on pod removal.
```

Comment:
:)
I was stuck to the english spelling (https://dictionary.cambridge.org/dictionary/english/behaviour)

Reply:
I wasn't aware both are correct :) Good to know :) 
Thanks for the reference!